docs/html/installing.rst

```diff
@@ -18,12 +18,14 @@ Just make sure to :ref:`upgrade pip <Upgrading pip>`.
 Installing with get-pip.py
 --------------------------
 
-To install pip, securely download `get-pip.py
-<https://bootstrap.pypa.io/get-pip.py>`_. [1]_::
+To install pip, securely download ``get-pip.py`` by following
+this link: `get-pip.py
+<https://bootstrap.pypa.io/get-pip.py>`_. [1]_. Alternatively, use ``curl``::
```

Comment:
I would move the `[1]_` earlier, like "securely [1]_ download" (and remove the trailing `.`). The result ends up looking similar to "setuptools [2]" and "user site [3]" from below, which is pretty good. :+1:

Reply:
Makes sense!